pubtools/_quay/command_executor.py

```diff
@@ -182,7 +182,9 @@ def _run_cmd(self, cmd, err_msg=None, tolerate_err=False, stdin=None):
         out, err = p.communicate(input=stdin)
 
         if p.returncode != 0 and not tolerate_err:
-            LOG.error("Command {0} failed with {1}".format(cmd, err))
+            LOG.error("Command {0} failed with the following error:".format(cmd))
+            for line in err.splitlines():
+                LOG.error(line)
```

Comment:
Can you add an indentation here? For example "    " could be enough to make it more readable in logs

Reply:
Added.